Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -313,6 +314,43 @@ function vtkResliceCursorWidget(publicAPI, model) {
     publicAPI.modified();
   };
 
+  publicAPI.rotatePlane = (rotatedPlaneName, rotateAngle, rotatedAxisName) => {
+    if (rotatedPlaneName === rotatedAxisName) {
+      return;
+    }
+    const normal = model.widgetState[`get${rotatedPlaneName}PlaneNormal`]();
+    const rotatedAxis = model.widgetState[`get${rotatedAxisName}PlaneNormal`]();
+    const newNormal = rotateVector(normal, rotatedAxis, rotateAngle);
+    model.widgetState[`set${rotatedPlaneName}PlaneNormal`](newNormal);
+    if (model.widgetState.getKeepOrthogonality()) {
+      let associatedRotatedPlaneName = 'X';
+      if (
+        (rotatedPlaneName === 'X' && rotatedAxisName === 'Y') ||
+        (rotatedPlaneName === 'Y' && rotatedAxisName === 'X')
+      ) {
+        associatedRotatedPlaneName = 'Z';
+      } else if (
+        (rotatedPlaneName === 'X' && rotatedAxis === 'Z') ||
+        (rotatedPlaneName === 'Z' && rotatedAxis === 'X')
+      ) {
+        associatedRotatedPlaneName = 'Y';
+      }
```

Comment:
I thought that you already have a convenient function that gives you the associated plane names. Don't you ?
If not, it might be useful to expose it as a convenient function.

Reply:
The method I have is a method which give me the associated axis (axisXinY or AxisYinZ,...) but not the plane. I'll add a function.